tests/pytests/test_config.py

```diff
@@ -1639,3 +1643,188 @@ def testDeprecatedConfigParamMessage():
+        else:
+            env.assertEqual(currentConfigDict[k], v, message=f'affectedConfig: {k}, changedConfig: {argName}')
+
+
+def testConfigIndependence_default():
+    """Test that changing one configuration value doesn't affect other configuration values"""
+    env = Env(noDefaultModuleArgs=True)
+
+    defaultConfigDict = getConfigDict(env)
+    for configName, argName, default, minValue, maxValue, immutable, clusterConfig in numericConfigs:
+        if immutable:
+            continue
+        if clusterConfig:
+            if not env.isCluster():
+                continue
+
+        # Test min value
+        checkConfigChange(env, configName, argName, minValue, defaultConfigDict)
+
+        # Test max value
+        checkConfigChange(env, configName, argName, maxValue, defaultConfigDict)
+
+        # Reset to default value
+        env.expect('CONFIG', 'SET', configName, default).ok()
+        currentConfigDict = getConfigDict(env)
+        env.assertEqual(currentConfigDict, defaultConfigDict)
+
+    for configName, argName, defaultValue, immutable, _ in booleanConfigs:
+        if immutable:
+            continue
+        # Test true (yes)
+        checkConfigChange(env, configName, argName, 'yes', defaultConfigDict)
+
+        # Test false (no)
+        checkConfigChange(env, configName, argName, 'no', defaultConfigDict)
+
+        # Reset to default value
+        env.expect('CONFIG', 'SET', configName, defaultValue).ok()
+        currentConfigDict = getConfigDict(env)
+        env.assertEqual(currentConfigDict, defaultConfigDict)
+
+def testConfigIndependence_min_values():
+    """Test that changing one configuration value doesn't affect other configuration values"""
+    env = Env(noDefaultModuleArgs=True)
+    # set all numeric configs to min value
+    for configName, argName, _, minValue, maxValue, immutable, clusterConfig in numericConfigs:
+        if immutable:
+            continue
+        if clusterConfig:
+            if not env.isCluster():
+                continue
+        env.expect('CONFIG', 'SET', configName, minValue).ok()
+    # set all boolean configs to false (no)
+    for configName, argName, _, immutable, _ in booleanConfigs:
+        if immutable:
+            continue
+        env.expect('CONFIG', 'SET', configName, 'no').ok()
+    minValueConfigDict = getConfigDict(env)
+
+    for configName, argName, _, minValue, maxValue, immutable, clusterConfig in numericConfigs:
+        if immutable:
+            continue
+        if clusterConfig:
+            if not env.isCluster():
+                continue
+
+        # Test max value
+        checkConfigChange(env, configName, argName, maxValue, minValueConfigDict)
+
+        # Reset to min value
+        env.expect('CONFIG', 'SET', configName, minValue).ok()
+        currentConfigDict = getConfigDict(env)
+        env.assertEqual(currentConfigDict, minValueConfigDict)
+
+    for configName, argName, _, immutable, _ in booleanConfigs:
```

Comment:
How is the check on Boolean configs different than the one in the previous test?